frontend/src/component/admin/users/AccessOverview/AccessOverview.tsx

```diff
@@ -104,19 +106,24 @@ export const AccessOverview = () => {
                 </PageHeader>
             }
         >
-            <StyledTitle>
-                Root permissions for role {rootRole?.name}
-            </StyledTitle>
-            <AccessOverviewTable permissions={overview?.root ?? []} />
-            <StyledTitle>
-                Project permissions for project {project} with project roles [
-                {projectRoles?.map((role: any) => role.name).join(', ')}]
-            </StyledTitle>
-            <AccessOverviewTable permissions={overview?.project ?? []} />
-            <StyledTitle>
-                Environment permissions for environment {environment}
-            </StyledTitle>
-            <AccessOverviewTable permissions={overview?.environment ?? []} />
+            <StyledAccessOverviewContainer>
+                <AccessOverviewAccordion permissions={overview?.root ?? []}>
+                    Root permissions for role {rootRole?.name}
+                </AccessOverviewAccordion>
+                <AccessOverviewAccordion permissions={overview?.project ?? []}>
+                    Project permissions
+                    {project
+                        ? ` for project ${project}${projectRoles?.length ? ` with project role${projectRoles.length !== 1 ? 's' : ''} ${projectRoles?.map((role: any) => role.name).join(', ')}` : ''}`
+                        : ''}
+                </AccessOverviewAccordion>
+                {environment && (
+                    <AccessOverviewAccordion
+                        permissions={overview?.environment ?? []}
+                    >
+                        Environment permissions for environment {environment}
```

Comment:
Maybe less redundant
```suggestion
                        Environment permissions for {environment}
```